lib/presentation/home/home_screen.dart

```diff
@@ -19,10 +19,13 @@ class _HomePageState extends State<HomePage> {
   @override
   void initState() {
     super.initState();
-    showOnboarding();
+    WidgetsBinding.instance?.addPostFrameCallback((_) {
+      checkAndMaybeShowOnboarding()
+          .then((_) => checkAndMaybeShowCaptivePage(context.router));
+    });
   }
 
-  Future<void> showOnboarding() async {
+  Future<void> checkAndMaybeShowOnboarding() async {

```

Comment:
❗ **Help requested:**
This method should call `context.router.push(const OnboardingRoute());` instead, but if I change it I always get an exception: `[Root Router] Router can not navigate to onboarding`.